benchopt/cli/helpers.py

```diff
@@ -38,6 +43,136 @@ def clean(benchmark, token=None, filename=None):
     rm_folder(cache_folder)
 
 
+def print_info(cls_list, env_name=None, dep=False):
+    """Print information for each element of input listed
+
+    Parameters
+    ----------
+    cls_list : list
+        List of objects (solvers or datasets) to print info from.
+    env_name : string | None
+        name of environment where to check for object availability.
+        If None or 'False', no check is made.
+    dep : boolean
```

Comment:
```suggestion
    dep : bool
```

Reply:
Fixed in 84c220c (argument is now useless and was removed, I forgot to do it when I removed the `--dep` flag)